tests/boards/nrf/ironside_se_boot_report/testcase.yaml

```diff
@@ -0,0 +1,19 @@
+common:
+  tags:
+    - drivers
+    - hwinfo
+  harness: console
+
+tests:
+  soc.nordic.ironside_se_boot_report:
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "err: 0"
+        - "version: 0x([0-9a-fA-F]+)"
+        - "extraversion: ([0-9a-fA-F]+)"
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp/iron
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp/iron
```

Comment:
NB: I had to get rid of integration_platforms to make the doc build green.

I don't know if testcase.yaml will have the same problem though.

https://github.com/nrfconnect/sdk-nrf/pull/21770/files

Reply:
Thanks for the heads up, will remove this if CI starts acting up.